File: DSA/Arartik_Tyagi_590011633/DAY_1/Problem1.java
import java.util.Scanner;

public class Problem1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number of elements");
        int a = sc.nextInt();
        int arr[] = new int[a];
        int sum = 0;
        System.out.println("Enter array elements");
        for (int i = 0; i < a; i++) {
            arr[i] = sc.nextInt();
            sum = sum + arr[i];
        }
        System.out.println("The sum of the array is " + sum);

        sc.close();
    }
}

File: DSA/Vaishnavi_590017838/Day_1/Problem1.cpp
#include <iostream>
using namespace std;
int main() {
    int arr[] = {12, 35, 1, 10, 34, 1};

    int n = sizeof(arr) / sizeof(arr[0]);
    int sum = 0;
    for(int i = 0; i < n; i++) {
        sum += arr[i];
    }
    cout << sum;
    return 0;
}
